terraform/southpark.tf

```diff
@@ -0,0 +1,170 @@
+provider "aws" {
+  version = "~> 2.0"
+  region  = "us-east-2"
+}
+
+
+#Create repo
+resource "aws_ecr_repository" "southpark_repo" {
+  name = "southpark_repo" # Naming my repository
+}
+
+# Providing a reference to our default VPC
+resource "aws_default_vpc" "default_vpc" {
+}
+
+# Providing a reference to our default subnets
+resource "aws_default_subnet" "default_subnet_a" {
```

Comment:
just curious, any reason for these many `subnets`?

Reply:
It is just because that's what I saw as standard in most docs

Reply:
As far as I know, you do it do have fault-tolerance. If you scale up you application you want them to be distributed across multiple availability zones. If one zone goes down then, at least a portion of you machines are still alive.

That apparently happened on AWS fall last year. A full availability zone in eu-central was gone for an hour or so. For my former employer, that meant, one third of the production-running machines where gone from one second to the next. If all running machines would have been in this availability zone, it would have been a serious disaster. But due to the split across multiple zones it went "quite smoothly".

Reply:
Anyhow, I am pretty sure, we don't need that here.